dynawo/sources/Models/Modelica/Dynawo/UsersGuide/NamingConventions.mo

```diff
@@ -55,16 +55,14 @@ annotation(preferredView = "info",
 <strong>Functions</strong>: start with lower case letter</li>
 </ul>
 
-<p>In addition to this, please follow the additional naming conventions bellow:</p>
+<p>In addition to this, please follow the additional naming conventions bellow (in that order):</p>
 
 <ul dir=\"auto\">
+<li><strong>Min/Max</strong>&nbsp;parameters/variables: end with&nbsp;<code>Min/Max</code></li><li>
+<strong>Start</strong> parameters/variables: end with <code>0</code></li>
 <li>
-<strong>Start</strong> parameters/variables: end with <code>Start</code></li>
-<li>
-<strong>Per-unit</strong> parameters/variables: end with <code>Pu</code></li>
-<li>
-<strong>Min/Max</strong> parameters/variables: end with <code>Min/Max</code></li>
-</ul>
+<strong>Per-unit</strong> parameters/variables: end with <code>Pu (always indicate \"in p.u\" in the comment along with the p.u base)</code></li>
```

Comment:
According to Wikipedia, the abbreviation of per-unit is pu without a dot.

Reply:
Let's keep p.u for the moment as it is the most used in the library. We could do a review later on when we have time.